src/prefect/utilities/feature_flags.py

```diff
@@ -0,0 +1,167 @@
+"""
+This module contains code related to feature flagging.
+
+For more guidance, read docs/feature_flags.md.
+"""
+from typing import Any, Iterable, List, Optional
+
+from flipper import Condition, FeatureFlagClient, MemoryFeatureFlagStore
+from flipper.bucketing.base import AbstractBucketer
+from flipper.flag import FeatureFlag
+
+from prefect import settings
+
+_client: Optional[FeatureFlagClient] = None
+
+
+def get_features_client() -> FeatureFlagClient:
+    global _client
+
+    if _client:
+        return _client
+
+    store = MemoryFeatureFlagStore()
+    _client = FeatureFlagClient(store)
+
+    return _client
+
+
+def create_if_missing(
+    flag_name: str,
+    is_enabled: bool = False,
+    client_data: Optional[dict] = None,
+    bucketer: Optional[AbstractBucketer] = None,
+    conditions: Optional[Iterable[Condition]] = None,
+    client: FeatureFlagClient = None,
+) -> Optional[FeatureFlag]:
+    """
+    Create a feature flag if a flag matching the given name does not
+    already exist.
+
+    Args:
+        flag_name: the name of the feature flag
+        is_enabled: the initial enabled/disabled state of the flag if
+                    this function creates it
+        client_data: arbitrary data that we should store with the flag
+        bucketer: an optional bucketer from the flipper.bucketing module, e.g.
+                  PercentageBucketer, to use when determining if the flag
+                  is enabled
+        conditions: an optional iterable of Conditions against which we will
```

Comment:
Does "Conditions" reference something that is a proper name? If not can we lowercase? I'm not familiar with the usage.

Reply:
`Condition` is a class name, so it should probably get the backtick treatment.